packages/web/components/templates/article/ArticleContainer.tsx

```diff
@@ -48,6 +54,21 @@ export function ArticleContainer(props: ArticleContainerProps): JSX.Element {
     updateFontSize(props.fontSize ?? 20)
   }, [props.fontSize])
 
+  // Load the highlights
```

Comment:
Did we need to move this out of the highlight layer?

Reply:
Hello @jacksonh, yeah, the highlights is being added asynchronously to the page and we need to know that it is ready before running the function to scroll to either the last read position or highlight with an id. We could use prop drilling to also achieve this. Another way we can go about this is if we take note of the position of the highlight parent and immediately scroll rather than using the highlight id

Reply:
[@jacksonh](https://github.com/jacksonh) Yeah, the highlights are being added asynchronously to the page and we need to know that they are ready before running the function to scroll to either the last read position or highlight with an id. We could use prop drilling to also achieve this. Another way we can go about this is if we take note of the position of the highlight parent and immediately scroll rather than using the highlight id

Reply:
Thanks. Personally I'd prefer prop pushing, but I'll test out this version as-is.